docs/backends/impala.md

```diff
@@ -10,6 +10,8 @@ intro: |
 exclude_backend_api: true
 ---
 
+# Impala
```

Comment:
this leads to a double title because this page is still using the `backends/template.md` file
```suggestion
```